src/rgw/rgw_sal_motr.cc

```diff
@@ -3656,6 +3857,18 @@ int MotrStore::set_buckets_enabled(const DoutPrefixProvider *dpp, vector<rgw_buc
   return 0;
 }
 
+uint64_t MotrStore::get_new_req_id()
+{
+  uint64_t req_id = ceph::util::generate_random_number<uint64_t>();
```

Comment:
This introduces a collision possibility for no good reason. Can a sequential atomic be used instead?

Reply:
@nikitadanilov, this id is used by S3 clients in responses. With sequential atomic, it's very probable that S3 clients will get the replies with the same ids from different rgw instances.

Reply:
But collision is still possible with random identifiers. Does a situation when a client gets the same identifiers from different instances lead to a problem?